BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -343,6 +384,23 @@ public final class ConvivaAnalytics: NSObject {
         buildDynamicContentMetadata()
     }
 
+    private func buildAdMetadata() -> [String: Any] {
+        let includedTags: Set<String> = [
```

Comment:
You don't carry over the asset name yet. Is this intentional?

Reply:
Actually this is a mistake, I missed that one. Fixed in 6fbe0f6